source/MaterialXView/ShaderValidators/ExceptionShaderValidationError.h

```diff
@@ -0,0 +1,48 @@
+#ifndef MATERIALX__EXCEPTIONSHADERVALIDATIONERROR_H
+#define MATERIALX__EXCEPTIONSHADERVALIDATIONERROR_H
+
+#include <MaterialXCore/Library.h>
+#include <string>
+
+namespace MaterialX
+{
+
+/// Error string list type
+using ShaderValidationErrorList = std::vector<std::string>;
```

Comment:
std:: namespace qualifier not needed for vector and string in MaterialX.

Reply:
Okay. Habit of using explicit namespace.